packages/stream-types-test/validation.ts

```diff
@@ -56,13 +56,31 @@ async () => {
 };
 
 async () => {
-  const s: Stream<string, number, boolean, boolean> =
+  const s: Stream<string> =
     prime(async function *generator() {
-    const n: number = yield 'A';
-    return false;
-  }());
+      yield 'A';
```

Comment:
To make sure the type yielded is really `undefined` and not `unknown`:
```suggestion
      const r: undefined = yield 'A';
```

Reply:
Addressed.